cmd/cmdutils.go

```diff
@@ -170,3 +173,46 @@ func printUnmountedStorage(client *occlient.Client, applicationName string) {
 	}
 	fmt.Println("")
 }
+
+// GetAppName returns application name
+func GetAppName(client *occlient.Client, project string) string {
+
+	if Application != "" {
+
+		_, err := application.Exists(client, project, Application)
+		if err != nil {
+			checkError(err, "")
+		}
+		return Application
+	}
+	applicationName, err := application.GetCurrent(project)
+	checkError(err, "unable to get current application")
+
+	return applicationName
+}
+
+// GetProjectName returns project name
+func GetProjectName(client *occlient.Client) string {
+	if Namespace != "" {
+		_, err := project.Exists(client, Namespace)
+		if err != nil {
+			checkError(err, "")
+		}
+		return Namespace
+	}
+
```

Comment:
Shouldn't we add a "GetCurrent" as well (similar to your GetAppName function?)

Reply:
added on the next line, check this https://github.com/redhat-developer/odo/pull/683/files#diff-28372c336478598898e80dfd05850c30R204